src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/InbreedingCoeff.java

```diff
@@ -72,13 +73,14 @@ public InbreedingCoeff(final File pedigreeFile){
         Utils.nonNull(vc);
         final GenotypesContext genotypes = getFounderGenotypes(vc);
         if (genotypes == null || genotypes.size() < MIN_SAMPLES || !vc.isVariant()) {
+            logger.warn("InbreedingCoeff will not be calculated; at least " + MIN_SAMPLES + " samples must have called genotypes");
             return Collections.emptyMap();
         }
         final Pair<Integer, Double> sampleCountCoeff = calculateIC(vc, genotypes);
         final int sampleCount = sampleCountCoeff.getLeft();
         final double F = sampleCountCoeff.getRight();
         if (sampleCount < MIN_SAMPLES) {
-            logger.warn("Annotation will not be calculated, must provide at least " + MIN_SAMPLES + " samples");
+            logger.warn("InbreedingCoeff will not be calculated for at least one position; at least " + MIN_SAMPLES + " samples must have called genotypes");
```

Comment:
Did you want separate "one shot" loggers for these 2 warnings? (ie., would you ever want both warnings to be emitted?)

Reply:
The first one is like "hey, you don't have enough samples in your cohort" and the second one is like "oops, there are a lot of no-calls at this site, so I'm not going to output a value here", so they should be mutually exclusive.